src/service/analytics.service.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { HelpConfig } from '../help-config';
+import { BBHelpAnalyticsService } from './analytics.service';
+
+const demoConfig: HelpConfig = {
+  productId: 'bbHelpTesting'
+};
+const DEVELOPMENT_KEY = '0e26030f769c1e630c59e1b3dec37957';
```

Comment:
Are these real keys?

Reply:
Good catch.  It would seem that they are. Thoughts on how we can inject these keys securely?  Perhaps through Travis?

Reply:
I guess having them hard-coded here isn't the end of the world... I mean they're going to be hard-coded after the build anyway, which would expose them.  I was mainly calling them out as it seemed weird to use the real ones in a spec file.  I would've expected something like `const DEVELOPMENT_KEY = 'example-development-key';`.

Reply:
Agreed.

Reply:
My question is why there's a real key in a spec file?  Shouldn't you be able to use a fake key since calling MixPanel would be mocked out in a unit test?

Reply:
I want to say it's because the variables for those keys are not attached to the class.  Since this class is attached to the window, though I can see the issue with the `getMixpanelKey` method returning the keys. it's still accessible.

Reply:
Should be resolved with this change: https://github.com/blackbaud/help-client/pull/39

Allows dependency injection to supply those keys.  They are isolated to one file that can be changed out as needed.  This also should isolate the keys so they are not directly accessibly from any methods on the window.